data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -275,3 +275,94 @@ def calculate_liquidable_debt(self, protocol_name: str = None) -> list:
                 )
 
         return result_data
+
+
+class HashstackV0DBLiquidableDebtDataHandler(BaseDBLiquidableDebtDataHandler):
+    """
+    A Hashstack_V0 handler that collects data from the DB,
+    computes the liquidable debt and stores it in the database.
+
+    :cvar AVAILABLE_PROTOCOLS: A list of all available protocols.
+    """
+
+    def __init__(
+            self,
+            loan_state_class: Type[State],
+            loan_entity_class: Type[LoanEntity],
+    ):
+        super().__init__()
+        self.state_class = loan_state_class
+        self.loan_entity_class = loan_entity_class
+
+    def initialize_loan_entities(self, state: State, data: dict = None):
+        """
+        Initializes the loan entities in a state instance.
+        :param state: State
+        :param data: dict
+        :return: None
+        """
+
+        for instance in data:
+            debt_category = self.db_connector.get_last_hashstack_loan_state(instance.user).debt_category
```

Comment:
double check before getting from method any attribure.
What is `self.db_connector.get_last_hashstack_loan_state(instance.user)` will return `None`